csharp/Platform.Data/ILinksExtensions.cs

```diff
@@ -16,13 +16,13 @@ namespace Platform.Data
     /// </summary>
     public static class ILinksExtensions
     {
-        public static TLink Create<TLink>(this ILinks<TLink, LinksConstants<TLink>> links) => links.Create(null);
+        public static TLink Create<TLink>(this ILinks<TLink, LinksConstants<TLink>> links) => links.Create(new List<TLink>());
```

Comment:
```suggestion
        public static TLink Create<TLink>(this ILinks<TLink, LinksConstants<TLink>> links) => links.Create(null);
```

Reply:
It is the main reason why I opened this PR. If I pass null I get exception

Reply:
Where exactly did you got the exception?